src/lib.rs

```diff
@@ -94,14 +94,21 @@
 
 #![no_std]
 
+// Some pac crates have fields specified in such a way that they are safe and other
+// have them unsafe (likely an SVD error that needs patching). Unsafe blocks for 
+// one device cause warnings for the safe devices. This disables that warning.
+#![allow(unused_unsafe)]
```

Comment:
I think this error should have been fixed by the latest PAC release, could you check that?

Reply:
There are still instances of mixed safe & unsafe registers. Examples:
stm32f100::tim1::aar::AAR_W::bits is safe
stm32f100::tim6::aar::AAR_W::bits is unsafe

I think the original comment still stands for now.

Reply:
Depending on how many such warnings there are, I would prefer having the `#[allow]` thing on the lines/functions where it happens. That way we don't miss actually useful warnings